packages/core/src/components/fields.tsx

```diff
@@ -60,8 +61,16 @@ export const Field = (props: FieldProps): JSX.Element | null => {
         if (props.onEnableChanged) {
             props.onEnableChanged();
         }
+    };
+
+    useLayoutEffect(() => {
```

Comment:
i think this should be combined, what if `hidden` is false and `disabled` is true? 

```
useLayoutEffect(() => {
   if (isHidden === undefined && isDisabled === undefined) {
      return;
   }
   setInputDisabled(isHidden || isDisabled);
}, [isHidden, isDisabled]);
```